Neos.Neos/Classes/Command/WorkspaceCommandController.php

```diff
@@ -178,13 +175,22 @@ public function createCommand(
         string $contentRepository = 'default'
     ): void {
         $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
+
+        $this->workspaceService->createPersonalWorkspace(
+            $contentRepositoryId,
+            NeosWorkspaceTitle::fromString($title ?? $workspace),
+            NeosWorkspaceDescription::fromString($description ?? ''),
+            WorkspaceName::fromString($baseWorkspace),
+            UserId::fromString($owner),
+        );
+
         $contentRepositoryInstance = $this->contentRepositoryRegistry->get($contentRepositoryId);
 
         if ($owner === '') {
             $workspaceOwnerUserId = null;
         } else {
             $workspaceOwnerUserId = UserId::fromString($owner);
-            $workspaceOwner = $this->userService->findByUserIdentifier($workspaceOwnerUserId);
+            $workspaceOwner = $this->userService->findUserById($workspaceOwnerUserId);
```

Comment:
as discussed with basti, we want to fix this to use `getUser` like in Neos8.3 instead so the more human readable name is used again not the persistence object identifier.